app/src/main/java/org/p2p/wallet/solend/ui/bottomsheet/SelectDepositTokenViewHolder.kt

```diff
@@ -0,0 +1,66 @@
+package org.p2p.wallet.solend.ui.bottomsheet
+
+import android.graphics.drawable.PictureDrawable
+import android.net.Uri
+import android.view.ViewGroup
+import android.widget.ImageView
+import com.bumptech.glide.Glide
+import com.bumptech.glide.RequestBuilder
+import com.bumptech.glide.request.RequestOptions
+import org.p2p.uikit.glide.SvgSoftwareLayerSetter
+import org.p2p.wallet.common.ui.recycler.adapter.BaseSelectionViewHolder
+import org.p2p.wallet.databinding.ItemSolendDepositTokenBinding
+import org.p2p.wallet.solend.model.SolendDepositToken
+import org.p2p.wallet.utils.formatToken
+import org.p2p.wallet.utils.scaleShort
+import org.p2p.wallet.utils.viewbinding.inflateViewBinding
+import java.math.BigDecimal
+
+class SelectDepositTokenViewHolder(
+    parent: ViewGroup,
+    private val binding: ItemSolendDepositTokenBinding = parent.inflateViewBinding(attachToRoot = false),
+    private val onItemClicked: (SolendDepositToken) -> Unit
+) : BaseSelectionViewHolder<SolendDepositToken>(binding.root, onItemClicked) {
+
+    companion object {
+        private const val IMAGE_SIZE = 48
+    }
+
+    private val requestBuilder: RequestBuilder<PictureDrawable> = Glide.with(binding.root.context)
+        .`as`(PictureDrawable::class.java)
+        .listener(SvgSoftwareLayerSetter())
+
+    override fun onBind(item: SolendDepositToken, selectedItem: SolendDepositToken?) {
```

Comment:
You should have only `SolendDepositToken.Active` here right?

Reply:
no not active